common/app/components/Nav/links.json

```diff
@@ -1,23 +1,46 @@
-[{
-  "content": "Community",
-  "isDropdown": true,
-  "links": [{
-    "content": "About",
-    "link": "/about"
-  }, {
-    "content": "Forum",
-    "link": "https://forum.freecodecamp.com/",
-    "target": "_blank"
-  }, {
-    "content": "Chat",
-    "link": "https://gitter.im/freecodecamp/home",
-    "target": "_blank"
-  }]
-},{
-  "content": "Map",
-  "link": "/map",
-  "isReact": true
-},{
-  "content": "Shop",
-  "link": "/shop"
-}]
+[
+  {
+    "content": "Community",
```

Comment:
@Bouncey we are getting rid of the map button completely once we move it over to the side. Here's what my prototype looks like: http://codepen.io/FreeCodeCamp/full/VKXyZJ